app/assets/stylesheets/content/work_packages/timelines/_timelines_header.sass

```diff
@@ -3,7 +3,7 @@
   width: 100%
   position: sticky
   top: 0
-  z-index: 1
+  z-index: 101
```

Comment:
This will cause the header to lay over the menu on MSEdge(mobile).

![bildschirmfoto 2018-12-13 um 10 02 26](https://user-images.githubusercontent.com/7457313/49927508-55b58180-febe-11e8-895b-7b9992c7e7c0.png)